### DISCUSSION TITLE
Nothing generated using import syntax

### DISCUSSION THREAD

**Discussion by aruvalcaba** [facelessuser/pymdown-extensions#2000] [Q&A] [answered]
--8<-- "wtf.md" or --8<-- "includes/wtf.md"

with both mkdocs serve and build, the above syntax just gets printed on the output page. I don't see any serve nor build errors. 

I have the following in my mkdocs.yml. Any help is greatly appreciated

theme:
  name: readthedocs
  logo: assets/logo-24.svg
  favicon: assets/favicon.png
  font:
    text: Roboto
  markdown_extensions:
    - pymdownx.snippets:
      base_path: includes
      check_paths: True
      throwException: True

**Comment by facelessuser** [comment]
I know nothing of your environment. If you were able to create a minimal, reproducible zip and provide it here, I may have the time to look at it, but right now, I have no idea from what little info you've provided.

**Reply by aruvalcaba** [reply]
Hi see attached zip folder. Tried a fresh new project, same problem.
[mkdocs.zip](https://github.com/facelessuser/pymdown-extensions/files/11159004/mkdocs.zip)

**Comment by facelessuser** [comment, marked as answer]
Please read MkDocs and Pymdown Extensions documentation carefully. This was a user configuration error. You did a couple of things incorrectly.

1. You indented Markdown extension options under the theme options. This will cause the Markdown extensions not to load as they will not be seen.

2. `throwExceptions` is not an option that is available in Snippets. This will cause the extension not to load as well.

3. This is more a YAML usage error, but you need to properly indent the YAML options for Snippets under the declaration for Snippets.

```yaml
site_name: My Docs

theme:
  name: readthedocs
  font:
    text: Roboto

markdown_extensions:
  - pymdownx.snippets:
      check_paths: True
      base_path: docs
```

**Reply by aruvalcaba** [reply]
Much appreciated for the help.
